src/modules/csm/state.py

```diff
@@ -33,6 +34,18 @@ def add_duty(self, included: bool) -> None:
         self.included += 1 if included else 0
 
 
+class StateDump(TypedDict):
+    attestationsOfValidators: dict[ValidatorIndex, AttestationsAccumulator]
```

Comment:
🐍 💼 ?

Reply:
🐫💼 for Jasón

Reply:
Let's better use snake_case in json files